Stripe/STPTelemetryClient.m

```diff
@@ -111,8 +100,9 @@ - (NSString *)language {
 - (NSString *)platform {
     struct utsname systemInfo;
     uname(&systemInfo);
-    NSString *deviceType = @(systemInfo.machine);
-    return deviceType ?: @"";
+    NSString *deviceType = @(systemInfo.machine) ?: @"";
```

Comment:
I was surprised by this too – boxing can also used for C strings
https://clang.llvm.org/docs/ObjectiveCLiterals.html#boxed-c-strings